tests/dags/providers/provider_api_scripts/test_phylopic.py

```diff
@@ -97,10 +104,10 @@ def test_get_meta_data_for_none_response():
 )
 
 
-def test_get_meta_data_correct():
-    with open(os.path.join(RESOURCES, "correct_meta_data_example.json")) as f:
-        r = json.load(f)
-    with patch.object(pp.delayed_requester, "get_response_json", return_value=r):
+def test_get_meta_data_correct(image_data):
+    with patch.object(
+        pp.delayed_requester, "get_response_json", return_value=image_data
+    ):
```

Comment:
Just a suggestion (because this was already not done in an optimal way) but we could use the [`monkeypatch` fixture helper](https://docs.pytest.org/en/6.2.x/monkeypatch.html) to create a fixture that automatically patches the image_data as the result of `get_response_json`.

Something like this I think?

```
@pytest.fixture(autouse=True)
def return_response_json(monkeypatch, image_data):
    monkeypatch.setattr(pp.delayed_requester, "get_response_json", value=lambda *args: image_data)
```

I don't remember if the lambda will work like that, it may be necessary to define the function separately as a `def` closure before monkeypatching.

Up to you whether to do this though :slightly_smiling_face:

Reply:
This will make the tests cleaner, but I think it is a change for a different PR :)